lib/email_messages.py

```diff
@@ -61,3 +62,28 @@ def email_course_error(request, exercise, message, exception=True):
         email_course_instance(instance, subject, body)
     except:
         pass
+
+
+def email_course_students(
+        instance: CourseInstance,
+        subject: str,
+        message: str,
+        audience: CourseInstance.ENROLLMENT_AUDIENCE=CourseInstance.ENROLLMENT_AUDIENCE.ALL_USERS
```

Comment:
Is `CourseInstance.ENROLLMENT_AUDIENCE` really accepted as a type here? I thought that the type is our own `Enum` class and `CourseInstance.ENROLLMENT_AUDIENCE` is just one instance of the `Enum`. I haven't tried this myself, so I am curious.

Our own A+ `Enum` is a normal class with no connection to the built-in enumerations in the newer Python versions.

The usual Python style for keyword arguments: 
* no whitespace when there is no static typing
* use whitespace when there static type is set
`audience: CourseInstance.ENROLLMENT_AUDIENCE = CourseInstance.ENROLLMENT_AUDIENCE.ALL_USERS,`

The comma in the end is also good to have when the parameters are already listed one per line.